sys/sysmon/include/sysmon/ThreadStatus.hpp

```diff
@@ -0,0 +1,91 @@
+///
+/// Copyright (C) 2022 COGIP Robotics association <cogip35@gmail.com>
+///
+/// This file is subject to the terms and conditions of the GNU Lesser
+/// General Public License v2.1. See the file LICENSE in the top level
+/// directory for more details.
+///
+///
+///
+/// @defgroup    sys_sysmon System monitoring
+/// @ingroup     sys
+/// @brief       System monitoring module
+///              Module used to monitor memory (thread stacks, heap and overall).
+///
+/// @{
+/// @file
+/// @brief       Public API for sysmon module
+///
+/// @author      Gilles DOFFE <g.doffe@gmail.com>
+///
+
+// System includes
+#include <cstddef>
+#include <string>
+
+// Project includes
+#include "MemoryStatus.hpp"
+#include "PB_ThreadStatus.hpp"
+#ifdef MODULE_UARTPB
+#include "uartpb/UartProtobuf.hpp"
+#endif
+
+#pragma once
+
+#ifndef THREADSTATUS_NAME_MAX_LENGTH
```

Comment:
We usually prefix module defines with the module name like `SYSMON_THREADSTATUS_NAME_MAX_LENGTH`.

Reply:
done